requirements.txt

```diff
@@ -3,7 +3,7 @@ cached-property==1.2.0
 dockerpty==0.4.1
 docopt==0.6.1
 enum34==1.0.4
-git+https://github.com/docker/docker-py.git@81d8caaf36159bf1accd86eab2e157bf8dd071a9#egg=docker-py
+git+https://github.com/mdaue/docker-py.git@d4a5bc4a86e9ab69368c80027439bca3ecc1e852#egg=docker-py
```

Comment:
Oops, I missed this, good catch @shin- 

It depends on https://github.com/docker/docker-py/pull/936. Looks like that needs a rebase